pkg/aws_az/environment/environment.py

```diff
@@ -0,0 +1,48 @@
+
+import os
+import pkg.types.types as types
+import pkg.maths.maths as maths
+
+#GetENV fetches all the env variables from the runner pod
+def GetENV(experimentDetails):
+	experimentDetails.ExperimentName =  os.getenv("EXPERIMENT_NAME", "aws-az-chaos")
+	experimentDetails.ChaosNamespace = os.getenv("CHAOS_NAMESPACE", "")
+	experimentDetails.EngineName = os.getenv("CHAOSENGINE", "")
+	experimentDetails.ChaosDuration = maths.atoi(os.getenv("TOTAL_CHAOS_DURATION", ""))
+	experimentDetails.ChaosInterval = os.getenv("CHAOS_INTERVAL", "")
+	experimentDetails.RampTime = maths.atoi(os.getenv("RAMP_TIME", ""))
```

Comment:
We should have a default value for these tunables.

Reply:
We need not to keep ramp time and all since it is  provided in CRs